service/worker/scheduler/workflow_test.go

```diff
@@ -1311,6 +1320,140 @@ func (s *workflowSuite) TestBackfillInclusiveStartEnd() {
 	)
 }
 
+func (s *workflowSuite) TestHugeBackfillAllowAll() {
```

Comment:
does the test framework guarantee that this test will never run concurrently with other tests in the file?

Reply:
Yes, because it's not marked t.Parallel(). We could probably arrange things to work in parallel with some refactoring, but it doesn't seem urgent

Reply:
I agree, it is not urgent to make the tests run in parallel. I just wanted to know what the framework will do since we modify global state in the test. Thank you for the explanation.